suite-common/transaction-cache-engine/src/tests/TransactionCacheEngine.test.ts

```diff
@@ -0,0 +1,137 @@
+import TrezorConnect from '@trezor/connect';
+
+import { TransactionCacheEngine } from '../TransactionCacheEngine';
+import { MemoryStorage } from '../MemoryStorage';
+import { accountBalanceHistoryResult as btcAccountBalanceHistoryResult } from './__fixtures__/btc';
+import { AccountBalanceHistory, AccountUniqueParams } from '../types';
+import { accountBalanceHistoryResult as rippleAccountBalanceHistoryResult } from './__fixtures__/xrp';
+
+jest.mock('@trezor/connect', () => {
```

Comment:
pls do not use custom connect mocks, use shared mock from test-utils,
changes example: https://github.com/trezor/trezor-suite/pull/10100

Reply:
Thanks, I will be doing follow-up where I plan to refactor tests even more so I will do it then